apps/src/applab/applab.js

```diff
@@ -161,6 +162,11 @@ function loadLevel() {
   for (var key in level.scale) {
     Applab.scale[key] = level.scale[key];
   }
+
+  if (Applab.makerlabEnabled) {
+    var BoardController = require('../makerlab/BoardController');
```

Comment:
Note: Because of how browserify works, BoardController will always be in the bundle (which may or may not be the intention). As such, I'd rather still have this require at the top of the file to make that more explicit.